FraudProtection/src/main/java/com/paypal/android/fraudprotection/PayPalDataCollector.kt

```diff
@@ -11,21 +13,26 @@ import lib.android.paypal.com.magnessdk.MagnesSource
  * Enables you to collect data about a customer's device and correlate it with a session identifier on your server.
  */
 class PayPalDataCollector internal constructor(
-    environment: PayPalDataCollectorEnvironment,
+    config: CoreConfig,
     private val magnesSDK: MagnesSDK,
     private val uuidHelper: UUIDHelper
 ) {
 
-    private val environment = getMagnesEnvironment(environment)
+    companion object {
+        fun getMagnesEnvironment(config: CoreConfig): Environment =
+            when (config.environment) {
+                com.paypal.android.corepayments.Environment.LIVE -> Environment.LIVE
+                com.paypal.android.corepayments.Environment.SANDBOX -> Environment.SANDBOX
+                com.paypal.android.corepayments.Environment.STAGING -> Environment.STAGE
```

Comment:
Do we have a follow up ticket to remove staging from Android for parity across platforms? I don't think we've ever used it.

Reply:
🤦 literally just saw you have another PR to do exactly this. Ignore me!

Reply:
Yeah this is a holdover from another PR. They were created around the same time we realized we need to remove staging.